docs/handbook/image-file-formats.rst

```diff
@@ -64,7 +64,8 @@ DDS
 
 DDS is a popular container texture format used in video games and natively supported
 by DirectX. Uncompressed RGB and RGBA can be read, and (since 8.3.0) written. DXT1,
-DXT3 (since 3.4.0) and DXT5 pixel formats can be read, only in ``RGBA`` mode.
+DXT3 (since 3.4.0) and DXT5 pixel formats can be read, only in ``RGBA`` mode. Since
+10.1.0, 8-bit color indexed files can be read as ``P`` mode images.
```

Comment:
Perhaps with `versionadded`?

```suggestion
DXT3 (since 3.4.0) and DXT5 pixel formats can be read, only in ``RGBA`` mode.

.. versionadded:: 10.1.0

8-bit color indexed files can be read as ``P`` mode images.
```

Or attempting to rewrite to avoid beginning a sentence with a numeral?

```suggestion
DXT3 (since 3.4.0) and DXT5 pixel formats can be read, only in ``RGBA`` mode.

.. versionadded:: 10.1.0

Files indexed with 8-bit color can be read as ``P`` mode images.
```

Reply:
From what I see in https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html, the tradition is for 'versionadded' to go below the new feature.

I've pushed a commit.

Reply:
The [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded) shows the text afterwards:

> The first argument must be given and is the version in question; you can add a second argument consisting of a _brief_ explanation of the change.
> 
> Example:
```rst
.. versionadded:: 2.5
   The *spam* parameter.
```


For example:

```rst
.. versionadded:: 3.4.0
   DXT3 images can be read in ``RGB`` mode and DX10 images can be read in ``RGB``
      and ``RGBA`` mode.
```

Gives:

<img width="730" alt="image" src="https://github.com/python-pillow/Pillow/assets/1324225/0874bd3a-50c4-4203-9586-1fedb4022f1f">

Which I think is clearer.